addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -391,6 +391,167 @@ options.registry.parallax = options.Class.extend({
     },
 });
 
+options.registry.facebook_page = options.Class.extend({
+    template: 'website.facebook_page_dialog',
+    xmlDependencies: ['/website/static/src/xml/website.facebook_page.xml'],
+
+    /**
+     * @override
+     */
+    start: function () {
+        var self = this;
+
+        this.$target.on('click', '.o_add_facebook_page', function (event) {
+            event.preventDefault();
+            self.fbPageOptions();
+        });
+        return $.when(this._setParams(), this._super.apply(this, arguments));
+    },
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Open dialog to configure facebook page options.
+     */
+    fbPageOptions: function () {
+        var self = this;
+
+        var $dialog = new Dialog(null, {
+            title: _t('Facebook page'),
+            $content: $(core.qweb.render("website.facebook_page_dialog", self.fbData)),
+            buttons: [
+                {text: _t('Save'), classes: 'btn-primary', close: true, click: function () {
+                    self.$target.empty();
+                    _.each(self.fbData, function (value, key) {
+                        self.$target.attr('data-' + key, value);
+                    });
+                    var actualWidth = self.$target.width();
+                    self.fbData.width = actualWidth > 500 ? 500 : actualWidth < 180 ? 180 : actualWidth;
+                    var iframeSrc = $.param.querystring('https://www.facebook.com/plugins/page.php', self.fbData);
+                    self.$target.append(_.str.sprintf('<iframe src="%s" width="%s" height="%s" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true"></iframe>', iframeSrc, self.fbData.width, self.fbData.height));
```

Comment:
Some of this code is a duplicate of the animation. On save, you just need to write the `data-...` then restart the animation with
```less
self.trigger_up('animation_start_demand', {
    editableMode: true,
    $target: self.$target,
});
```